pims/bioformats.py

```diff
@@ -93,7 +94,7 @@ def _download_jar(version='5.7.0'):
 
 def _maybe_tostring(field):
```

Comment:
Under what circumstances do we *not* want to convert `field` to a string?

If I remove the "maybe" from this function and just return `str(field)` every time, all of the tests pass. 
Only 10 of the 160 bioformats tests trigger the `if hasattr(field, 'toString')` clause at some point, these cause failures if not converted to strings. But I don't see a case where we don't want to convert to a string. Is this a gap in the test coverage, or can this function be replaced with plain calls to `str()`?

Reply:
It looks like `_maybe_tostring()` admitted the possibility that its input might be a Python int, float, etc. But it looks like that is not and never was a possibility if the value came from Jpype. So I have removed it! Travis does not test with `jpype1=0.6` but I just did and it still works.